src/cockpit/389-console/src/lib/database/localPwp.jsx

```diff
@@ -116,90 +113,106 @@ class CreatePolicy extends React.Component {
     }
 
     render() {
+        let helper_text = "Required field";
+        if (this.props.invalid_dn) {
+            helper_text = "Invalid DN";
+        }
+
         return (
             <div>
-                <Form className="ds-margin-left" horizontal>
-                    <Row>
-                        <h4>Create A New Local Password Policy</h4>
-                    </Row>
-                    <Row className="ds-margin-top-lg">
-                        <Col componentClass={ControlLabel} sm={3}>
+                <Form className="ds-margin-left ds-margin-top-xlg" isHorizontal>
+                    <h4 className="ds-center">
+                        Create A New Local Password Policy
```

Comment:
A minor comment. Maybe it makes sense to set default 'suggested' values. It may be something copied from the global password policy which the user can modify then.

Reply:
Lets look at this in a separate ticket.  There are other issues with global plocy and default values as well.  We can look at them both in a separate issue